HelpSource/Classes/DetectSilence.schelp

```diff
@@ -77,3 +78,15 @@ Task({
 )
 
 ::
+
+section::DetectSilence and multiple inputs
+
+DetectSilence has a non-obvious interaction with multiple inputs. This code
```

Comment:
I'd avoid saying 'non-obvious' here; this behavior could be perfectly predictably to someone who hasn't used DetectSilence before and is only familiar with multi-channel expansion. "Note that this line" should be enough.